resources/italian.yml

```diff
@@ -1,8 +1,8 @@
 name: Towny
 version: 0.90
 language: italian
-author: Leomixer17
-website: 'http://townyadvanced.github.io/'
+author: Leonardo_DGS
```

Comment:
The file looks good but you should keep Leomixer17's name on the author line as well.

Reply:
I am Leomixer17

Reply:
I guess that resolves that, ha.